test/StandaloneHydratorPluginManagerTest.php

```diff
@@ -0,0 +1,116 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-hydrator for the canonical source repository
+ * @copyright Copyright (c) 2018 Zend Technologies USA Inc. (https://www.zend.com)
+ * @license   https://github.com/zendframework/zend-hydrator/blob/master/LICENSE.md New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace ZendTest\Hydrator;
+
+use Closure;
+use PHPUnit\Framework\TestCase;
+use ReflectionProperty;
+use Zend\Hydrator;
+use Zend\Hydrator\StandaloneHydratorPluginManager;
+
+use function array_pop;
+use function sprintf;
+
+class StandaloneHydratorPluginManagerTest extends TestCase
+{
+    public function setUp()
```

Comment:
IMHO we should keep the visibility from the parent class, so it could be `protected` instead.

Reply:
We use public visibility pretty consistently everywhere for the setUp and tearDown methods. I'm not going to change it here.